package.json

```diff
@@ -400,7 +400,7 @@
     }
   ],
   "engines": {
-    "vscode": "^1.44.0"
+    "vscode": "^1.46.0"
```

Comment:
Need to update the vscode version to get access to EnvironmentVariableCollection.